### ISSUE TITLE
pScheduler batch processing logic

### ISSUE THREAD

**Issue by epcjr** [perfsonar/pscheduler#1201]
It would be a useful feature to add some logic to the pScheduler batch processor that made some tests contingent on other tests passing. This would be useful in the situation where you wanted to auth and get a DHCP lease from a WiFi access point before running latency, throughput, etc.

**Comment by mfeit-internet2** [comment]
## TODO

- [ ] Merge into whatever version will contain this (nominally 5.0 unless we do a 4.5).

## Release Notes

Added the ability for the batch processor to use a jq transform on job results to determine whether or not to proceed with the next job.

**Comment by mfeit-internet2** [comment]
TODO:

- [ ] Add `continue-if` to docs.
